workos/utils/request_helper.py

```diff
@@ -8,14 +9,19 @@
 REQUEST_METHOD_POST = "post"
 REQUEST_METHOD_PUT = "put"
 
+QueryParameterValue = Union[str, int, bool, None]
+QueryParameters = Dict[str, QueryParameterValue]
```

Comment:
Mapping?

Reply:
We set items after assignment on `QueryParameters` objects (e.g. params["test"]).